poster/styles/styles.scss

```diff
@@ -1,63 +1,477 @@
-.u-text_main {
+// utility //
+
+.u-text_details {
     font-size: .7rem;
     margin-left: 1rem;
     margin-right: 1rem;
 }
 
+.u-remove-list {
```

Comment:
Just a nitpick here, but "remove" seems to give the wrong idea; maybe `u-unstyled-list` or something similar ?